### PR TITLE
[RFR][1LP]Add pytest.skip in configure_vmware_console for vsphere65

### PR DESCRIPTION
**Pull request by nzr-zts**
Purpose 
=================
Add pytest.skip for vsphere65, VNC consoles are unsupported on VMware ESXi 6.5 and later, which causes error _"DropdownItemDisabled: Item "VM Console" of dropdown "Access" is disabled"_

{{pytest: -v cfme/tests/ssui/test_ssui_myservice.py -k test_vm_console[vsphere65-nested-order_catalog_item_in_ops_ui0-ViaSSUI]}}

### CODE REVIEW

**Comment by kedark3** [comment]
LGTM. As PR #6523 is moving the fixtures to separate file, we might need to move this condition to that file as well.

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 1]
```diff
@@ -68,7 +68,10 @@ def test_retire_service(appliance, setup_provider, context, order_catalog_item_i
 def configure_vmware_console_for_test(appliance, provider):
```
Comment (psav) [inline cfme/tests/ssui/test_ssui_myservice.py]: **REQUIRED:** Let's do this as an uncollect please, skip implies the test should run and is valid for the configuration, it's not.

Comment (nzr-zts) [reply cfme/tests/ssui/test_ssui_myservice.py]: Sure, thank you for the hint.

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 2]
```diff
@@ -73,6 +73,8 @@ def configure_vmware_console_for_test(appliance, provider):
 
 @pytest.mark.parametrize('context', [ViaSSUI])
 @pytest.mark.parametrize('order_catalog_item_in_ops_ui', [['console_test']], indirect=True)
+@pytest.mark.uncollectif(lambda provider: provider.key >= 'vsphere65',
```
Comment (psav) [inline cfme/tests/ssui/test_ssui_myservice.py]: **REQUIRED:** Can we change this ```provider.key``` to be ```provider.one_of(VMwareProvider) and provider.version < 6.5```

**Comment by dajoRH** [comment]
Lint report for commit 2ad2311381c58bf20c01974d4be275c449af854a:

`cfme/tests/ssui/test_ssui_myservice.py`:
- :bangbang: Line 77:43: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**